### ISSUE TITLE
WyzeCam V3 on Firmware 4.36.11.8391 timeout

### ISSUE THREAD

**Issue by tadcrazio** [mrlt8/docker-wyze-bridge#1132]
Using Docker Wyze bridge Current version: 2.3.17 as an add-on in Home assistant. 

I have a few cameras on Firmware 4.36.11.7095 and they seem to work just fine but the 2 cameras I have updated to  4.36.11.8391 I keep getting a timeout when previously they were working without issues. 

[garage] [-13] IOTC_ER_TIMEOUT
[WyzeBridge] ☁️ Fetching 'cameras' from the Wyze API...
[front-porch-cam] [-13] IOTC_ER_TIMEOUT
[WyzeBridge] 💾 Saving 'cameras' to local cache...
[WyzeBridge] 🎉 Connecting to WyzeCam V3 - GARAGE on 10.0.0.9
[WyzeBridge] 🎉 Connecting to WyzeCam V3 - Front Porch Cam on 10.0.0.182
[WyzeBridge] ⏰ Timed out connecting to GARAGE.
[WyzeBridge] ⏰ Timed out connecting to Front Porch Cam.
[WyzeBridge] 🎉 Connecting to WyzeCam V3 - GARAGE on 10.0.0.9
[WyzeBridge] 🎉 Connecting to WyzeCam V3 - Front Porch Cam on 10.0.0.182
[WyzeBridge] ⏰ Timed out connecting to GARAGE.
[WyzeBridge] ⏰ Timed out connecting to Front Porch Cam.
[WyzeBridge] 🎉 Connecting to WyzeCam V3 - GARAGE on 10.0.0.9
[WyzeBridge] 🎉 Connecting to WyzeCam V3 - Front Porch Cam on 10.0.0.182
[WyzeBridge] ⏰ Timed out connecting to GARAGE.
[WyzeBridge] ⏰ Timed out connecting to Front Porch Cam.
[WyzeBridge] 🎉 Connecting to WyzeCam V3 - GARAGE on 10.0.0.9
[WyzeBridge] 🎉 Connecting to WyzeCam V3 - Front Porch Cam on 10.0.0.182
[front-porch-cam] [-13] IOTC_ER_TIMEOUT
[WyzeBridge] ⏰ Timed out connecting to GARAGE.
[WyzeBridge] 🎉 Connecting to WyzeCam V3 - GARAGE on 10.0.0.9
[WyzeBridge] 🎉 Connecting to WyzeCam V3 - Front Porch Cam on 10.0.0.182
[garage] [-13] IOTC_ER_TIMEOUT

I have tried restarting my router, power cycling the cameras, add/removing sub streams, adding/removing audio. I've restarted the hardware. I just can't seem to get these two cameras to connect which happened recently after updating the firmware on the cameras. The only thing I can see different between these 2 and the others are the firmware as noted above. 

I've turned on debug logging but I don't see anything of use in there.

**Comment by leatherlipps** [comment]
I have this same issue.  I just opened another Issue for the V2 cameras that upgrade to 4.9.9.3006 firmware as well.  They are now getting same timeout

**Comment by tadcrazio** [comment]
Had a v3 pancam shared with me on firmware 4.50.4.8409 and same issue there.

**Comment by theBasher91** [comment]
Anyone resolve this besides doing a firmware rollback?

**Comment by letrain02** [comment]
All mine are working fine on 4.36.11.8391

**Comment by tadcrazio** [comment]
> All mine are working fine on 4.36.11.8391

V3's? 
I notice snapshots sometimes work, but can't get the streams. updated to latest version too with same issue

**Comment by glenviewjeff** [comment]
Checking in on the status of these errors; I am having no success lately pulling streams. All of my firmwares had been updated to whatever is current as of today. I hadn't been using Wyze bridge actively but now that I'm looking into frigate, I noticed no streams are working and the log has IOTC_ER_TIMEOUTs for what I believe is every camera, various camera models including V3s, OGs, and the newest doorbells.

Even snapshots seem to update extremely sporadically and unpredictably, once every day or two, and won't update when refreshed.

I'm on a Google Wifi mesh network; a camera reset didn't make a difference. I tried @letrain02 's idea of the fresh dataset to no avail. I even set it to static IP address, though that really makes no sense to me since the IP address allocation with this system stays static anyhow.

**Comment by PCjabber** [comment]
> I hadn't been using Wyze bridge actively but now that I'm looking into frigate, I noticed no streams are working and the log has IOTC_ER_TIMEOUTs for what I believe is every camera, various camera models including V3s, OGs, and the newest doorbells.
> 
> Even snapshots seem to update extremely sporadically and unpredictably, once every day or two, and won't update when refreshed.

Unfortunate timing to jump back in 😔...Wyze seems to have changed something on their end in the past few days. There are a few other threads about this issue (not sure which one is most active, and I haven't found a solution yet; I'm using Docker on PhotonOS): #1432 #1433 #1438 #1435 

As far as I can tell, my cams have not firmware upgraded anytime recently. I have three Cam v3 and one Pan v2. One of the Cam v3 is still working with the Wyze-Bridge (running older FW 4.36.10.4054). The others stopped working ~3 days ago (two Cam v3 running FW 4.36.13.0416, and a Pan v2 running FW 4.49.13.0653). All cams live-stream fine through the Android app.

**Comment by glenviewjeff** [comment]
> > I hadn't been using Wyze bridge actively but now that I'm looking into frigate, I noticed no streams are working and the log has IOTC_ER_TIMEOUTs for what I believe is every camera, various camera models including V3s, OGs, and the newest doorbells.
> > Even snapshots seem to update extremely sporadically and unpredictably, once every day or two, and won't update when refreshed.
> 
> Unfortunate timing to jump back in 😔...Wyze seems to have changed something on their end in the past few days. There are a few other threads about this issue (not sure which one is most active, and I haven't found a solution yet; I'm using Docker on PhotonOS): [#1432](https://github.com/mrlt8/docker-wyze-bridge/issues/1432) [#1433](https://github.com/mrlt8/docker-wyze-bridge/issues/1433) [#1438](https://github.com/mrlt8/docker-wyze-bridge/issues/1438) [#1435](https://github.com/mrlt8/docker-wyze-bridge/issues/1435)
> 
> As far as I can tell, my cams have not firmware upgraded anytime recently. I have three Cam v3 and one Pan v2. One of the Cam v3 is still working with the Wyze-Bridge (running older FW 4.36.10.4054). The others stopped working ~3 days ago (two Cam v3 running FW 4.36.13.0416, and a Pan v2 running FW 4.49.13.0653). All cams live-stream fine through the Android app.

Thanks for letting me know. I rolled back firmware on a pan V3 camera to 4.50.4.7252 and it was subsequently able to stream. So I'm a bit confused. I also happen to be running my cameras on a different subnets (the wifi guest network.) It looks like there are some temporary fixes available involving new config files and/or builds on the threads you referred me to, thank you. I may wait a few days for this to settle down before I spend more time looking into frigate.

**Comment by PCjabber** [comment]
> Thanks for letting me know. I rolled back firmware on a pan V3 camera to 4.50.4.7252 and it was subsequently able to stream. So I'm a bit confused. I also happen to be running my cameras on a different subnets (the wifi guest network.) It looks like there are some temporary fixes available involving new config files and/or builds on the threads you referred me to, thank you. I may wait a few days for this to settle down before I spend more time looking into frigate.

Happy to help, and glad you got yours working again! I've tried the "host" network mode for the Docker container with no change, unfortunately (actually, worse, until I realized the host firewall was blocking traffic, oops!). My cameras are all on the same subnet, but connected through multiple WiFi APs (one of which is uplinked to the LAN via a set of wireless 60GHz radios). Perhaps interesting -- the one that's still working in Blue Iris (& through the Bridge web UI) is connected to the AP that's physically plugged into the same switch as my Blue Iris NVR and the host running the Docker Wyze Bridge...but it also has the oldest firmware.

I'm trying to roll back firmware on my non-working Cam v3's, but can't get them to go into the "status light will appear solid purple" state [that Wyze references](https://support.wyze.com/hc/en-us/articles/360031490871-How-to-flash-your-Wyze-Cam-firmware-manually). Holding down the setup button while plugging it in, the light goes red very quickly. Is it possible that I can't downgrade to the specific version I'm trying? Just wondering if you ran into any issues while downgrading. Thanks!

**Comment by glenviewjeff** [comment]
Yes, I had the same problem. When I tried it on my existing 128gb cards that have camera data on them, the camera wouldn't downgrade and the led never turned purple. This is a v3 pan. Wyze [(recommends?) 32gb sdcards for upgrades](https://support.wyze.com/hc/en-us/articles/360031490871-How-to-flash-your-Wyze-Cam-firmware-manually). Fortunately I had one on hand. I also did not do a quick format but a full FAT32 format as at least one page suggested. I also crossed my fingers, performed an incantation while burning incense stick inserted into an eye of newt, and then held my breath. 

> > Thanks for letting me know. I rolled back firmware on a pan V3 camera to 4.50.4.7252 and it was subsequently able to stream. So I'm a bit confused. I also happen to be running my cameras on a different subnets (the wifi guest network.) It looks like there are some temporary fixes available involving new config files and/or builds on the threads you referred me to, thank you. I may wait a few days for this to settle down before I spend more time looking into frigate.
> 
> Happy to help, and glad you got yours working again! I've tried the "host" network mode for the Docker container with no change, unfortunately (actually, worse, until I realized the host firewall was blocking traffic, oops!). My cameras are all on the same subnet, but connected through multiple WiFi APs (one of which is uplinked to the LAN via a set of wireless 60GHz radios). Perhaps interesting -- the one that's still working in Blue Iris (& through the Bridge web UI) is connected to the AP that's physically plugged into the same switch as my Blue Iris NVR and the host running the Docker Wyze Bridge...but it also has the oldest firmware.
> 
> I'm trying to roll back firmware on my non-working Cam v3's, but can't get them to go into the "status light will appear solid purple" state [that Wyze references](https://support.wyze.com/hc/en-us/articles/360031490871-How-to-flash-your-Wyze-Cam-firmware-manually). Holding down the setup button while plugging it in, the light goes red very quickly. Is it possible that I can't downgrade to the specific version I'm trying? Just wondering if you ran into any issues while downgrading. Thanks!

**Comment by joaldes** [comment]
I was trying to get the RTSP firmware installed today, before I saw this widespread problem.  I can't get the RTSP firmware, nor an earlier version of firmware to flash on the camera.  I tried using a 32GB card, doing a full format, and even using the SD Card formatter (https://www.sdcard.org/downloads/formatter/) to no avail.  

Otherwise, it just stays solid red, has a blue blink for a microsecond, then goes to the 'ready to connect' process.  Annoying.

**Comment by glenviewjeff** [comment]
Did you hold the reset button while you plugged in the camera? Also, note
that the instructions are wrong, there's no folder on the SD card, just the
file in the root folder of the card.

**Comment by joaldes** [comment]
Yes, I held the 'Setup' button while I plugged the camera back in, and simply added the firmware file directly to the card, not inside any folder.  I tried this process a dozen times already - both with the RTSP firmware and the previous version firmware.

**Comment by glenviewjeff** [comment]
I just took a look at that formatting tool you used; it says it uses
something called SDA's SD File System. I've never heard of this and can't
find any useful information about what this means exactly, and what the
filesystem "really" is, but I wouldn't use this tool under any
circumstances personally. You can just do a Windows format and make sure 
to use FAT32 per the instructions. How long did you hold the button down? I
think it took at least 10-15 seconds, also differing from the instructions,
again on a pan cam v3. Hold it for at least thirty seconds just to be safe.
The LED definitely turned purple when I finally got it to work.

**Comment by joaldes** [comment]
It was a tool referenced by other people who had this problem - and formatting with the tool solved the issue.  

I tried different ways of formatting the cards (32GB and 128GB).

I held the button down for quite a while, and the camera just seemed like it was bootlooping.  It would keep going to the 'ready to connect' stage, without ever getting the purple light

**Comment by glenviewjeff** [comment]
The only thing I can think of at this point is to try a different model of 32GB sdcard. I read at least one person mention they used a Wyze branded one to be "safe," though mine was either a Transcend 10 or Silicon Power brand (I could check if it's really important). 

> It was a tool referenced by other people who had this problem - and formatting with the tool solved the issue.
> 
> I tried different ways of formatting the cards (32GB and 128GB).
> 
> I held the button down for quite a while, and the camera just seemed like it was bootlooping. It would keep going to the 'ready to connect' stage, without ever getting the purple light

**Comment by PCjabber** [comment]
Just to circle back on this, I ended up using the workaround described [here by FeatherKing](https://github.com/mrlt8/docker-wyze-bridge/discussions/1430#discussioncomment-12295564) in #1430 -- using the `ipvlan` network driver, configured for the same subnet as the Wyze cameras (on a network interface that's connected to the same Layer-2 network). 
Everything is working properly now, without downgrading camera firmware. 

Thanks again @glenviewjeff for your feedback!
